dashboard/app/models/pd/application/active_application_models.rb

```diff
@@ -3,9 +3,8 @@ module Application
     module ActiveApplicationModels
       include ApplicationConstants
 
-      APPLICATION_CURRENT_YEAR = YEAR_22_23
-      APPLICATION_NEXT_YEAR = YEAR_23_24
-      APPLICATION_CURRENT_YEAR_SHORT = YEAR_22_23_SHORT
```

Comment:
APPLICATION_CURRENT_YEAR_SHORT isn't used anywhere, so I removed it and the associated short year strings